doc/gui/examples/controls/column_name_styling.py

```diff
@@ -0,0 +1,21 @@
+# Example for column name styling
+
+from taipy.gui import Gui
+import pandas as pd
+from taipy.gui import Markdown
+
+# Sample data in DataFrame format
+df = pd.DataFrame({
+    "Name": ["Alice", "Bob", "Charlie"],
+    "Age": [25, 30, 35],
+    "Job": ["Engineer", "Doctor", "Artist"]
```

Comment:
I don't get why you cannot have a column name with space in it so that we can show the classname transformation ?